skipruntime-ts/server/src/rest.ts

```diff
@@ -23,7 +23,7 @@ export function createRESTServer(runtime: SkipRuntime): express.Express {
       );
       const [name, value] = reactiveResponseHeader(data);
       res.set(name, value);
-      res.status(200).json({});
+      res.status(200).json();
```

Comment:
I think you can simply remove the .json() call

Reply:
Yes but I need sendStatus instead of status.